docs/subnets/subnet-upgrade.md

```diff
@@ -95,9 +95,9 @@ Refer to the previous section on how to make sure node is healthy and connected
 
 If you don't get the expected result, you can stop the `AvalancheGo`, examine and follow closely step-by-step of the above. You are free to remove files under `~/avalanche-node/plugins`, however, you should keep in mind that removing files is to remove an existing VM binary. You must put the correct VM plugin in place before you restart AvalancheGo.
 
-## Changing Subnet Configuration
+## Network Upgrades
 
-Sometimes you need to do a Subnet upgrade to change the configuration and rules under which the Subnet operates. As before we will discuss `subnet-evm` upgrades, as that is the most commonly used Subnet VM.
+Sometimes you need to do a network upgrade to change the genesis configuration and rules under which the Subnet operates. As before we will discuss `subnet-evm` upgrades, as that is the most commonly used Subnet VM.
```

Comment:
Can we update this to make it clear that we are changing the chain configuration and EVM rule set as opposed to changing the genesis configuration? It's a bit of a misnomer to say we're changing the genesis configuration since it's set in the genesis and not actually changeable.